README.md

```diff
@@ -65,147 +65,221 @@ TechInfo lookup is looking at Toyota's Techinfo site (payment required, minimum
 
 For vehicles that do work, here's a very rough guide. Please send in pull requests to fix or amend it.
 
+### Preamble
```

Comment:
pedantic but i like double newlines after markdown headers.

Reply:
I knew I forgot something... Let me review the existing doc and match the formatting

Reply:
Did my best to match what's already there. PTAL

Reply:
Happy to change the formatting if I missed something. Please feel free to nitpick at it.